govcd/edgegateway_lb_service_monitor.go

```diff
@@ -0,0 +1,187 @@
+package govcd
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+// CreateLBServiceMonitor synchronously creates a load balancer service monitor based on mandatory fields
+// and returns created object with all fields (including ID) populated or an error.
+func (eGW *EdgeGateway) CreateLBServiceMonitor(lbMonitorConfig *types.LBMonitor) (*types.LBMonitor, error) {
+	if err := validateCreateLBServiceMonitor(lbMonitorConfig); err != nil {
+		return nil, err
+	}
+
+	httpPath, err := eGW.getProxiedEdgeEndpoint("/loadbalancer/config/monitors")
+	if err != nil {
+		return nil, fmt.Errorf("could not get Edge Gateway API endpoint: %s", err)
+	}
+	// We expect to get http.StatusCreated or if not an error of type types.NSXError
+	resp, err := eGW.client.ExecuteRequestHTTPCodeOrTypedError(http.StatusCreated, httpPath, http.MethodPost, "application/xml", "error creating load balancer service monitor: %s", lbMonitorConfig, &types.NSXError{})
```

Comment:
so NSX API isn't returning any `Tasks` ?

Reply:
No. These calls are synchronous. Response returns ID of newly created object.